### PR TITLE
17727- Fix Continuation In flow bug

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#17727

*Description of changes:*

Bug - On first load if COOP is selected, Check Name button was shown. If I then choose a different business type like "Limited Company", then select "Cooperative Association"  Check Name button was shown

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-695-o7bbii7l.web.app

File: src/components/new-request/search.vue [hunk 1]
```diff
@@ -401,12 +401,6 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
   }
 
   get showCheckNameButton (): boolean {
-    // Conditional for Continuation In Flow.
```
Comment (ketaki-deodhar) [inline src/components/new-request/search.vue:404]: Removed it. This was merge issue. Same logic is on line 436.

File: src/components/new-request/search.vue [hunk 2]
```diff
@@ -434,8 +428,9 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
 
     // Conditional for Continuation In Flow.
     if (this.isContinuationIn) {
-      if (this.getEntityTypeCd && this.isNamedCompany) return true
+      if (this.getEntityTypeCd && !this.isNumberedEntityType) return true
       if (this.getEntityTypeCd && !this.isNumberedEntityType && this.isSociety) return true
+      if (this.getEntityTypeCd && this.isNamedCompany) return true
```
Comment (severinbeauvais) [inline src/components/new-request/search.vue:433]: Yup. Amalgamations had several bugs related to this (that I fixed), especially when changing between entity types.

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by leodube-aot — APPROVED** [review]

**Pull request merged**